checkbox-ng/checkbox_ng/urwid_ui.py

```diff
@@ -691,11 +691,11 @@ def interrupt_dialog(host):
         ('start', 'dark green,bold', 'black'),
     ]
     choices = [
-        _("Cancel the interruption and resume the session (ESC)"),
-        _("Kill test in progress and move on to next"),
-        _("Disconnect the master (Same as CTRL+C)"),
-        _("Stop the checkbox slave @{}".format(host)),
-        _("Abandon the session on the slave @{}".format(host)),
+        _("Nothing, continue testing (ESC)"),
+        _("Kill test in progress and move on to the next"),
+        _("Exit but let the service test run continue (CTRL+C)"),
```

Comment:
We have [that card](https://warthogs.atlassian.net/browse/CHECKBOX-656) to figure out what naming to use, and it looks like we're gonna go with `controller` and `testbed`.

In this case, using `Disconnect the controller (same as Ctrl+C)` would make sense. What do you think?

Reply:
+1.
Maybe even: `Disconnect the controller, but let the session continue (same as Ctrl+C)`
Also the original `Exit but let the service test run continue` is difficult to read.